concent_api/conductor/service.py

```diff
@@ -125,10 +125,10 @@ def _store_blender_crop_script_parameters(blender_parameters: Any) -> BlenderCro
         resolution_y=blender_parameters['resolution'][1],
         samples=blender_parameters['samples'],
         use_compositing=blender_parameters['use_compositing'],
-        borders_x_min=blender_parameters['borders_x'][0],
-        borders_x_max=blender_parameters['borders_x'][1],
-        borders_y_min=blender_parameters['borders_y'][0],
-        borders_y_max=blender_parameters['borders_y'][1],
+        borders_x_min=str(blender_parameters['borders_x'][0]),
```

Comment:
What happens when `blender_parameters['borders_x'][0]` is `None`?

Reply:
It won't get to that point, we have validations for types in `message_handlers.py`. But:
```
../service.py:133: in _store_blender_crop_script_parameters
    blender_crop_script_parameters.full_clean()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <BlenderCropScriptParameters: BlenderCropScriptParameters object>
exclude = ['borders_y_max'], validate_unique = True

    def full_clean(self, exclude=None, validate_unique=True):
        """
            Calls clean_fields, clean, and validate_unique, on the model,
            and raises a ``ValidationError`` for any errors that occurred.
            """
        errors = {}
        if exclude is None:
            exclude = []
        else:
            exclude = list(exclude)
    
        try:
            self.clean_fields(exclude=exclude)
        except ValidationError as e:
            errors = e.update_error_dict(errors)
    
        # Form.clean() is run even if other validation fails, so do the
        # same with Model.clean() for consistency.
        try:
            self.clean()
        except ValidationError as e:
            errors = e.update_error_dict(errors)
    
        # Run unique checks, but only for fields that passed validation.
        if validate_unique:
            for name in errors.keys():
                if name != NON_FIELD_ERRORS and name not in exclude:
                    exclude.append(name)
            try:
                self.validate_unique(exclude=exclude)
            except ValidationError as e:
                errors = e.update_error_dict(errors)
    
        if errors:
>           raise ValidationError(errors)
E           django.core.exceptions.ValidationError: {'borders_y_max': ["'None' value must be a decimal number."]}
```